pkg/controllers/cspi-controller/controller.go

```diff
@@ -92,12 +98,24 @@ func NewCStorPoolInstanceController(
 	eventBroadcaster.StartRecordingToSink(&typedcorev1.EventSinkImpl{Interface: kubeclientset.CoreV1().Events("")})
 	recorder := eventBroadcaster.NewRecorder(scheme.Scheme, corev1.EventSource{Component: poolControllerName})
 
+	// Instantiate the zcmdExecutor to execute zpool/zfs commands
+	zcmdExecutor := zcmd.NewZcmd()
+
+	// Instantiate the operations config
+	operationsConfig := zpool.NewOperationsConfig().
+		WithKubeClientSet(kubeclientset).
+		WithOpenEBSClient(clientset).
+		WithRecorder(recorder).
+		WithZcmdExecutor(zcmdExecutor)
```

Comment:
We do not need this operationsConfig... Maybe I should have explained a bit better.
We already have these properties in CStorPoolInstanceController.
Let us just use this CStorPoolInstanceController object.

Reply:
Let us have the methods of CStorPoolInstanceController instead of `operationsConfig`